cdap-ui/app/directives/group-side-panel/group-side-panel.less

```diff
@@ -60,17 +135,27 @@ my-side-panel {
         }
       }
 
-
       .item-body {
         padding: 5px;
```

Comment:
Should there be a max-height on the item-body. What happens if more #of plugins (source) gets added in. It should automatically fix a height for the item-body and let it scroll when #of items grows more . Thoughts?